BSP/Simulator/Src/BSP_Timer.c

```diff
@@ -0,0 +1,44 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include "Timer.h"
+
+
+int update_Time(int row)
```

Comment:
We want to use specific int types that use the minimum required space for whatever variable you're using.

Reply:
Also, this function should update all timers. And its primary purpose should be to call whatever function the timer was initialized to call. The whole point is to have a timer that calls some function every so often (like an interrupt).